nodejs/eks/package.json

```diff
@@ -11,15 +11,15 @@
     "homepage": "https://pulumi.com",
     "repository": "https://github.com/pulumi/pulumi-eks",
     "dependencies": {
-        "@pulumi/aws": "^5.1.2",
-        "@pulumi/kubernetes": "^3.0.0",
+        "@pulumi/aws": "5.10.0",
+        "@pulumi/kubernetes": "3.20.0",
```

Comment:
@danielrbradley why this version and not 3.21.4 (the latest before v3.22.0 made SSA the default)?

Reply:
Almost: it was 3.20.**2** 😉

Reply:
Trying bumping to the latest version as tests seem to be stuck having pinned the version. It's possible that our test entirely ignore the lock file too 🤦

Reply:
Well, moving to the latest versions breaks a whole load of tests ... so going back to the pinned versions again 🤷